monitor/census.go

```diff
@@ -855,6 +876,36 @@ func InitCensus(nodeType NodeType, version string) {
 			TagKeys:     baseTags,
 			Aggregation: view.Count(),
 		},
+
+		// Metrics for AI jobs
+		{
+			Name:        "ai_models_requested",
+			Measure:     census.mAIModelsRequested,
+			Description: "Number of AI models requested over time",
+			TagKeys:     append([]tag.Key{census.kPipeline, census.kModelName}, baseTags...),
+			Aggregation: view.Count(),
+		},
+		{
+			Name:        "ai_request_latency_score",
+			Measure:     census.mAIRequestLatencyScore,
+			Description: "AI orchestrator request latency score",
+			TagKeys:     append([]tag.Key{census.kPipeline, census.kModelName}, baseTagsWithOrchInfo...),
+			Aggregation: view.LastValue(),
+		},
+		{
+			Name:        "ai_request_price",
+			Measure:     census.mAIRequestPrice,
+			Description: "AI request price per unit",
+			TagKeys:     append([]tag.Key{census.kPipeline, census.kModelName}, baseTags...),
```

Comment:
It may help to track the price by orchestrator. What do you think?
```suggestion
			TagKeys:     append([]tag.Key{census.kPipeline, census.kModelName}, baseTagsWithOrchInfo...),
```

Reply:
@eliteprox I first had that however from the Ecosystem side I would simply want to know the average price per pipeline which I can calculate based on this value. @ad-astra-video, @mikezupper what do you think. Would you use the info per orchestrator?

Reply:
@eliteprox let's put it right back -> https://github.com/livepeer/go-livepeer/pull/3087/commits/bd5b047694a8b25643f8865a436761a4f0a8c7f6. @ad-astra-video feel free to let me know if you think this will cause to much metrics to be generated 👍🏻.

Reply:
On second thought, I don't think there is a "normal" use case for having the orchestrator specific pricing. I think it would be helpful as part of a network reporting tool similar to stronk.rocks. 

I agree that it could generate too much data. One option would be to implement something similar to the `-PerStreamMetrics` flag, setting the baseflags conditionally, for example:

https://github.com/livepeer/go-livepeer/blob/b7181d425c64095da04a5c7e277173885afc0ab5/monitor/census.go#L383-L387

Reply:
I did add the orchestrator label to the pricing as you suggested (see https://github.com/livepeer/go-livepeer/blob/a92e9df2a6f32c2eb6a72ef3d162daf9944887f5/monitor/census.go#L899). However wasn't sure if this will lead to to much data. Curious to hear what @ad-astra-video thinks 🤔.